platform/o.apache.commons.lang3/build.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+-->
+<project name="platform/o.apache.commons.lang3" default="build" basedir=".">
+    <import file="../../nbbuild/templates/projectized.xml"/>
+    <target name="jar" depends="-define-FileCRC32Calculator">
+        <FileCRC32Calculator file="external/commons-lang3-3.8.1.jar" property="o.apache.commons.lang3.crc32" />
+        <jar jarfile="${cluster}/${module.jar}">
```

Comment:
I guess this follows a pattern copied from somewhere else, right?

Reply:
Copied from another apache commons lib module.